src/vanta_ledger/routes/projects.py

```
@@ -0,0 +1,28 @@
+from fastapi import APIRouter, Depends, HTTPException
+from typing import List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)